CONTRIBUTING.md

```diff
@@ -89,24 +89,62 @@ The changelog should follow the following convention:
 
 **README**
 
-The `README.md` should cover the following territory:
+The `README.md` should follow this template:
 
 ```
+# (package name) &middot; (badges)
+
 ## Description
 (high-level summary that will be accessible to product/UX/Test, plus link to Storybook, explain the different cases)
+
 ## When to use this component
 (including where it is currently used)
-## When to not use this component
+
+## When not to use this component
 (if this is not known, can leave the section empty)
+
+## Installation
+`npm install (package-name)`
+
+## Props
+(a table of all the component's props, following the template below)
+
+| Argument  | Type                | Required | Example         |
+|-----------|---------------------|----------|-----------------|
+| prop-name | description of type | Yes/No   | `example input` |
+
+## Usage
+(An code block containing an example usage of the component, including importing)
+
 ## Accessibility notes
 (ensuring they make sense to non-technical audiences)
+
 ## Roadmap
 (what we have planned for this component)
+
 ## Additional notes
 (link to any relevant ADRs)
+
+## Contributing
+
+Psammead is completely open source. We are grateful for any contributions, whether they be new components, bug fixes or general improvements. Please see our primary contributing guide which can be found at [the root of the Psammead respository](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md).
+
+### [Code of Conduct](https://github.com/BBC-News/psammead/blob/latest/CODE_OF_CONDUCT.md)
+
+We welcome feedback and help on this work. By participating in this project, you agree to abide by the [code of conduct](https://github.com/BBC-News/psammead/blob/latest/CODE_OF_CONDUCT.md). Please take a moment to read it.
+
+### License
+
+Psammead is [Apache 2.0 licensed](https://github.com/BBC-News/psammead/blob/latest/LICENSE).
 ```
 
-In addition, the `README.md` should also point to Psammead's [CODE_OF_CONDUCT](https://github.com/BBC-News/psammead/blob/latest/CODE_OF_CONDUCT.md), [CONTRIBUTING](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md) guide, and [LICENSE](https://github.com/BBC-News/psammead/blob/latest/LICENSE).
+When linking to other pages from a component's README, we advise you do so only by absolute URL, and avoid the use of relative paths. Component READMEs are displayed in a variety of places, including NPM and Storybook, and this helps ensure the intended meaning is preserved.
+
+If you wish to exclude one or more sections of this template, we advise you do so by simply commenting out the heading, like so:
+
+```
+<!-- ## Roadmap -->
+```
```

Comment:
Why? Feels a bit unclean 🤔

Reply:
I wanted to try & encourage consistency. One of the most irritating parts of doing this work was the fact that "adding a usage section to every README" was muddied with the fact that not every "usage" section was called "usage". It feels we should have a clear pattern for this, which will make it more reliable for users & for linking to specific subheadings.

Reply:
Oh yeah, you jogged my memory -- I think I'd briefly looked into #147 at the time, & considered that this would allow us to assert than every package still contains the string `## Roadmap` without requiring a section header with an empty body.